src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothLeAdapterImpl.cs

```diff
@@ -394,6 +393,7 @@ internal IList<BluetoothLeServiceData> GetScanResultServiceDataList(BluetoothLeS
 
             Log.Info(Globals.LogTag, "Count of ServiceDataList: " + _serviceListCount);
 
+            IList<BluetoothLeServiceData> _list = new List<BluetoothLeServiceData>();
```

Comment:
`_` prefix not need on local variable
```suggestion
            IList<BluetoothLeServiceData> list = new List<BluetoothLeServiceData>();
```